core/neat/mixins/_grid-container.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+
+/// Creates Neat grid container with clearfix.
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-container;
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: auto;
+///     margin-right: auto;
+///   }
+///
+///   .element::after {
+///     clear: both;
+///     content: "";
+///     display: block;
+///   }
+
+@mixin grid-container($grid: $neat-default-grid) {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
I agree that the `margin` styles may do more harm than good. I'm okay with it just being a clearfix wrapper. It's really all we need.